<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleParaSetPage</class>
 <widget class="QWidget" name="VehicleParaSetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>380</y>
     <width>91</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border:3px solid;
border-color:rgb(230, 230, 230);
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>MC1车轮径1(mm):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>设定值   </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  12px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>(范围580~660)mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wheelConfirm">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border:3px solid;
border-color:rgb(230, 230, 230);
</string>
   </property>
   <property name="text">
    <string>轮径设定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wheelSetValue_MC1_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>MC1_1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  24px ;
color: rgb(255, 255, 255);
background-color: rgb(178, 178, 178);
border:2px solid rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wheelSetValue_MC1_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>MC1_2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  24px ;
color: rgb(255, 255, 255);
background-color: rgb(178, 178, 178);
border:2px solid rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>MC1车轮径2(mm):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wheelSetValue_MC2_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>MC2_1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  24px ;
color: rgb(255, 255, 255);
background-color: rgb(178, 178, 178);
border:2px solid rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>T车轮径1(mm):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>T车轮径2(mm):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wheelSetValue_MC2_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>MC2_2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  24px ;
color: rgb(255, 255, 255);
background-color: rgb(178, 178, 178);
border:2px solid rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wheelSetValue_T_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>T_2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  24px ;
color: rgb(255, 255, 255);
background-color: rgb(178, 178, 178);
border:2px solid rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>MC2车轮径2(mm):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>MC2车轮径1(mm):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wheelSetValue_T_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>T_1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  24px ;
color: rgb(255, 255, 255);
background-color: rgb(178, 178, 178);
border:2px solid rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleWheelset_label_24">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>车辆编号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_numberConfirm">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border:3px solid;
border-color:rgb(230, 230, 230);
</string>
   </property>
   <property name="text">
    <string>车号设定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Number">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>Number</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  24px ;
color: rgb(255, 255, 255);
background-color: rgb(178, 178, 178);
border:2px solid rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="VehicleParaSet_label_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  12px;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>(范围00~99)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string> 当前值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>设定值   </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string> 当前值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>设定值   </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string> 当前值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_wheelCurValue_MC1_1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>600</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_wheelCurValue_MC1_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>600</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_wheelCurValue_T_1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>600</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_wheelCurValue_T_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>600</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_wheelCurValue_MC2_1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>600</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_wheelCurValue_MC2_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>600</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_curNumber">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  16px ;
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
